src/main/java/core/basesyntax/strategy/impl/HandlerStrategyImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy.impl;
+
+import static core.basesyntax.Main.OPERATION_HANDLER_MAP;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.HandlerStrategy;
+import core.basesyntax.strategy.OperationHandler;
+
+public class HandlerStrategyImpl implements HandlerStrategy {
+
+    @Override
+    public OperationHandler get(FruitTransaction.Operation operation) {
```

Comment:
Look, you call "OPERATION_HANDLER_MAP" from Main class. But Main is just entry point to launch you code local. It is not part of you program (it's just for you). 
Create here Map (as instance variable ) and just initialized it through constructor and in Main implement your real map.

```suggestion
private final Map<FruitTransaction.Operation, OperationHandler> nameOfMap;

class constructor(Map<FruitTransaction.Operation, OperationHandler> nameOfMap) {
this.nameOfmap = nameOfmap; 
}
    public OperationHandler get(FruitTransaction.Operation operation) {
```